src/backend/marsha/core/permissions/token.py

```diff
@@ -116,13 +115,8 @@ def has_permission(self, request, view):
         """
         return (
             request.resource
-            and models.Playlist.objects.filter(
-                Q(pk=view.get_object_pk())
-                & (
-                    Q(videos__id=request.resource.id)
-                    | Q(documents__id=request.resource.id)
-                ),
-            ).exists()
+            and request.resource.id == view.get_object_pk()
+            and models.Playlist.objects.filter(pk=request.resource.id).exists()
```

Comment:
Here you are using `pk=` and line 173 you are using `id=`. Both are good IMO, we should probably choose the one we use everywhere.